### ISSUE TITLE
Elastic transformations

### ISSUE THREAD

**Issue by lext** [Oulu-IMEDS/solt#11]
Support of elastic transforms for keypoints, masks and images is needed

**Comment by hoanghng** [comment]
This looks usable https://www.kaggle.com/bguberfain/elastic-transform-for-data-augmentation

**Comment by lext** [comment]
yes and no. The does not extend easily to the keypoints. Some work needs to be done...

**Comment by aisosalo** [comment]
How about https://github.com/uoip/transforms/blob/master/transforms.py#L1004-L1054

**Comment by lext** [comment]
This is not enough as the transform has to support the warping of the **keypoints** as well.

**Comment by aisosalo** [comment]
Keypoints could be given as an image :) of some type and then after the transformation extracted from the distorted image?

**Comment by lext** [comment]
I am reopening this issue for release 0.3, as release 0.2 has already quite many important fixes and new features.
